frontend/services/popupNotifications.service.js

```diff
@@ -3,69 +3,83 @@
 
   angular
     .module("base")
-    .service("popupNotifications", popupNotifications);
+    .factory("popupNotifications", popupNotifications);
 
   popupNotifications.$inject = ["Alertify"];
 
   function popupNotifications(Alertify) {
-    var vm = this;
-    vm.notifyAlert = notifyAlert;
-    vm.notifyConfirm = notifyConfirm;
-    vm.notifySuccess = notifySuccess;
-    vm.notifyStandart = notifyStandart;
-    vm.notifyError = notifyError;
-    vm.notifyPersistent = notifyPersistent;
-    vm.notifyNewMsg = notifyNewMsg;
+    let service = {
+      notifyAlert: notifyAlert,
+      notifyConfirm: notifyConfirm,
+      notifySuccess: notifySuccess,
+      notifyStandard: notifyStandard,
+      notifyError: notifyError,
+      persistentSuccess: persistentSuccess,
+      persistentStandard: persistentStandard,
+      persistentError: persistentError
+    };
+
+    return service;
 
     function notifyAlert(alertMsg) {
-      alertMsg = alertMsg || "Hello in Farm Apps";
+      alertMsg = alertMsg || "Alert Dialog!";
       Alertify.alert(alertMsg);
     }
 
     function notifyConfirm(confirmMsg) {
       confirmMsg = confirmMsg || "Are you sure?";
-      Alertify.set({buttonReverse: true});
-      Alertify.confirm(confirmMsg).then(
-        function onOk() {
-          Alertify.success("You've clicked Ok");
-        },
-        function onCancel() {
-          Alertify.error("You've clicked Cancel")
+
+      Alertify.set({
```

Comment:
Method should get ok and cancel labels. In case when they (or one of them) are defined set them.

Also check whether it is necessary to restore default buttons names.